docs/result.md

```diff
@@ -227,3 +227,159 @@ File: document.json
       }
     }
 }
+```
+##Client report failure
+
+a POST request is invoked by the client to the server. It returns a JSON file with information like client_id, job_id, and Message to return failure endpoint in the server. The client can return a failure do to many reasons such as:
+* 400: Bad request
+* 401: Unauthorized
+* 404: Not found
+* 408: Request Timeout
+* 480: Temporarily Unavailable
+
+
+## JSON file definition
+
+* `client_id`: A unique identifier of the client returning the result.
+* `job_id`: ID of the current job object.
+
+* `Massage`: a string list contains the URL of failure job and the description of the error.
+
+
+## Examples for failure jobs
+### Docket:
+
+1- bad docketID:
```

Comment:
I would use * or 1. a single tab for order listing.
Since we are using * instead of numbers, I think that will match the pattern.
But if you choose to use numbers then I would do 1. and single tab